gulpfile.js

```diff
@@ -219,6 +219,16 @@ const mappings = {
     'dir': 'powerbiembedded/lib',
     'source': 'arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json'
   },
+  'rdbms.mysql': {
+    'packageName': 'azure-arm-rdbms',
+    'dir': 'rdbmsManagement/lib/mysql',
+    'source': 'arm-rdbms/2017-04-30-preview/swagger/Rdms.MySQL.json'
+  },
```

Comment:
Agree with you. I can see where that will complicate any effort at regenerating all. I saw the Python SDK has an autogenerated client for these new services already, generated from the private repo and though it would be ok. But again... I totally see your point. Do you know if it's actually ok for us to go about moving that swagger spec to the public repo? Or do you know who we can check with?